design/host-cgroups.md

```diff
@@ -0,0 +1,230 @@
+- [Host cgroup management](#host-cgroup-management)
+  - [Introduction](#introduction)
+  - [`SandboxCgroupOnly` enabled](#sandboxcgrouponly-enabled)
+    - [What does Kata do in this configuration?](#what-does-kata-do-in-this-configuration)
+    - [Why create a Kata-cgroup under the parent cgroup?](#why-create-a-kata-cgroup-under-the-parent-cgroup)
+    - [Improvements](#improvements)
+  - [`SandboxCgroupOnly` disabled (default, legacy)](#sandboxcgrouponly-disabled-default-legacy)
+    - [What does this method do?](#what-does-this-method-do)
+      - [Impact](#impact)
+  - [Summary](#summary)
+
+# Host cgroup management
+
+## Introduction
+
+Kata Containers runs over two layers of cgroups, the first layer is in the
+guest where only the workload is placed, the second layer is in the host and it
+will be explained on this document.
+
+```
+$ docker run -dt --runtime=kata-runtime nginx
```

Comment:
Could we split this block into something like:

> For example, when running this command:
> ```sh
> $ docker run -dt --runtime=kata-runtime nginx
> ```
>
> The following cgroups are created:
>
> << ASCII graphics awesomeness goes here >>

Reply:
actually.... I want to just remove this part of the intro...